android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-photo-view')
```

Comment:
i was thinking the same thing these past few weeks. It's React Native - so it's inevitable that more and more dependencies are going to be added from here on out - that's one of the reasons why I love RN so much. 

But just like you said, we all usually try limiting dependencies in our applications generally to reduce bloat. I really _don't know_ how much application load times/performance is effected in a native application to be honest. However I of course would probably agree with most people and limit dependencies to only those we particulary need, even if a native application isn't as affected as much.